### DISCUSSION TITLE
1.0 Task Priority

### DISCUSSION THREAD

**Discussion by babyfish-ct** [babyfish-ct/jimmer#818] [General]
We are currently working on document reconstruction and handling the shit about MySQL's `rewriteBatchedStatements`, which will take some time. 

After these things are completed, we will move towards 1.0

Currently, most users think that there are two urgent things

1. Accept non-ORM returns and let subqueries serve as base tables for `from` and `join`

2. ORM inheritance

Please tell us your priority, `1` or `2`?

**Comment by pot-mot** [comment]
1

**Comment by Enaium** [comment]
1

**Comment by TokgoRonin** [comment]
1

**Comment by luqiuzhi** [comment]
2

**Comment by Neptrue-Lin** [comment]
1

**Comment by belovaf** [comment]
1
I think inheritance can be workarounded much easier.

**Comment by SWQXDBA** [comment]
1
因为事实上jimmer已经支持一种继承了 mappedsuperclass
基于join的继承可以用one to one代替
而且orm的多态实现和使用上都很复杂，意义可能不大
如果有3 我投auto-ddl

**Reply by luqiuzhi** [reply]
对于复杂的业务系统来说orm层面支持继承还是很有必要的。但是auto-ddl确实是个关键的功能，看了半天文档都没找到这个特性🤨

**Comment by draco1023** [comment]
2

**Comment by FunnyCavy** [comment]
1

**Comment by phenzler** [comment]
1
And don't forget the package name change discussed in 
https://github.com/babyfish-ct/jimmer/discussions/903
